CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
-# 0.7.4
+### 4.0.0
+Major update with breaking changes
```

Comment:
If we put this in master, how will users designate prior botkit versions for mock to use?

Reply:
it is gonna be the same version naming as for botkit